src/lsp/status-bar.ts

```diff
@@ -0,0 +1,64 @@
+import * as vscode_lsp from 'vscode-languageclient/node';
+import * as vscode from 'vscode';
+
+export enum LspStatus {
+  Stopped = 'Stopped',
+  Starting = 'Starting',
+  Running = 'Running',
+  Downloading = 'Downloading',
+  Failed = 'Failed',
+  Unknown = 'Unknown',
+}
+
+export const lspClientStateToStatus = (state: vscode_lsp.State): LspStatus => {
+  switch (state) {
+    case vscode_lsp.State.Stopped: {
+      return LspStatus.Stopped;
+    }
+    case vscode_lsp.State.Starting: {
+      return LspStatus.Starting;
+    }
+    case vscode_lsp.State.Running: {
+      return LspStatus.Running;
+    }
+  }
+};
+
+export const updateStatusBar = (
+  item: vscode.StatusBarItem,
+  status: LspStatus,
+  extraInfo?: string
+) => {
+  switch (status) {
+    case LspStatus.Stopped: {
+      item.text = '$(circle-outline) clojure-lsp';
+      item.tooltip = 'Clojure-LSP is not active, click to get a menu';
```

Comment:
```suggestion
      item.tooltip = 'Clojure-lsp is not active, click to get a menu';
```

Reply:
Would you mind changing this and other occurrences so that we're more consistent with the casing in the docs and elsewhere?

Reply:
Done in bcefbe7a